### PR TITLE
fix: versioned docs

### PR DESCRIPTION
**Pull request by ashnamehrotra**
**What this PR does / why we need it**:
Fixes versioned docs PR by updating for v0.4.0

**Which issue(s) this PR fixes** *(optional, using `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when the PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by sozercan — DISMISSED** [review]

File: Makefile [hunk 1]
```diff
@@ -252,4 +252,4 @@ version-docs:
 		-w /docs \
 		-u $(shell id -u):$(shell id -g) \
 		node:${NODE_VERSION} \
-		sh -c "yarn install --frozen-lockfile && yarn run docusaurus docs:version ${NEWVERSION}"
\ No newline at end of file
+		sh -c "yarn install && yarn run docusaurus docs:version ${NEWVERSION}"
```
Comment (sozercan) [inline Makefile]: why did we remove `--frozen-lockfile`?

File: docs/yarn.lock [hunk 2]
```diff
@@ -1569,18 +1569,18 @@
     react-helmet-async "*"
     react-loadable "npm:@docusaurus/react-loadable@5.5.2"
 
-"@docusaurus/plugin-content-blog@2.1.0":
-  version "2.1.0"
-  resolved "https://registry.yarnpkg.com/@docusaurus/plugin-content-blog/-/plugin-content-blog-2.1.0.tgz#32b1a7cd4b0026f4a76fce4edc5cfdd0edb1ec42"
-  integrity sha512-xEp6jlu92HMNUmyRBEeJ4mCW1s77aAEQO4Keez94cUY/Ap7G/r0Awa6xSLff7HL0Fjg8KK1bEbDy7q9voIavdg==
+"@docusaurus/plugin-content-blog@2.0.1":
```
Comment (sozercan) [inline docs/yarn.lock]: looks like we are downgrading?

File: Makefile (hunk 1 above)
Comment (ashnamehrotra) [reply Makefile]: I was getting this error to update the lockfile: "error Your lockfile needs to be updated, but yarn was run with `--frozen-lockfile`."

File: docs/yarn.lock (hunk 2 above)
Comment (ashnamehrotra) [reply docs/yarn.lock]: I think this is related to the --frozen-lockfile update?

File: Makefile [hunk 3]
```diff
@@ -252,4 +252,4 @@ version-docs:
 		-w /docs \
 		-u $(shell id -u):$(shell id -g) \
 		node:${NODE_VERSION} \
-		sh -c "yarn install --frozen-lockfile && yarn run docusaurus docs:version ${NEWVERSION}"
```
Comment (sozercan) [inline Makefile:255]: @ashnamehrotra looks like `preset-classic` got stuck on `2.0.1` for some reason. can we update to `2.1.0` in https://github.com/Azure/eraser/blob/main/docs/package.json#L18?

and then run `yarn` again (which should update the lockfile) and let's re-add the `--frozen-lockfile`

File: yarn.lock [hunk 4]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (sozercan) [inline yarn.lock]: @ashnamehrotra looks like this is extra, we can delete this file

File: docs/yarn.lock (hunk 2 above)
Comment (sozercan) [reply docs/yarn.lock]: did you run `yarn` again in `docs` folder? it should update these to 2.1.0 i think?

Comment (ashnamehrotra) [reply docs/yarn.lock]: accidentally ran it outside of docs! should be fixed now

**Review by sozercan — APPROVED** [review]
Thank you! LGTM

**Pull request merged**